src/themes/default/pages/Compare.vue

```diff
@@ -18,20 +18,29 @@
                 <li class="compare__features-item">
                   {{ $t('SKU') }}
                 </li>
-                <li v-for="(attr, index) in all_comparable_attributes" :key="index" class="compare__features-item">
+                <li
+                  v-for="(attr, index) in all_comparable_attributes"
+                  :key="index"
+                  class="compare__features-item"
+                >
                   {{ attr.default_frontend_label }}
                 </li>
               </ul>
             </div>
             <div class="compare__products-wrapper">
               <ul class="compare__products-columns">
-                <li v-for="(product, index) in items" :key="index" class="compare__product" data-testid="comparedProduct">
+                <li
+                  v-for="(product, index) in items"
+                  :key="index"
+                  class="compare__product"
+                  data-testid="comparedProduct"
+                >
                   <div class="compare__top-info">
                     <div class="check" />
                     <product-tile class="col-md-12 collection-product" :product="product" />
-                    <span class="compare__remove" @click="removeFromCompare(product)">
+                    <!-- <span class="compare__remove" @click="removeFromCompare(product)">
```

Comment:
Don't leave commented out code. This will stay forever as everybody will be afraid to remove it because it probably has a reason to stay like that.